packages/fxa-settings/src/pages/InlineRecoverySetupFlow/container.tsx

```diff
@@ -129,16 +129,15 @@ export const InlineRecoverySetupContainer = ({
   const verifyTotpHandler = useCallback(async () => {
     try {
       await authClient.completeTotpSetup(
-        signinRecoveryLocationState!.sessionToken,
-        { service: serviceName }
+        signinRecoveryLocationState!.sessionToken
```

Comment:
I'm surprised to the service option missing? Should this be addressed in another issue?